src/gui/moduleControlWidgetFuncs.cpp

```diff
@@ -50,15 +45,14 @@ ModuleControlWidget::ModuleControlWidget(DissolveWindow *dissolveWindow, Module
 
         // Add a new KeywordsWidget, set it up, and add it to the stack
         auto *w = new KeywordsWidget();
-        w->setUp({groupName, sections}, keywordMap, dissolve_.coreData());
+        w->setUp(section, dissolve_.coreData());
         connect(w, SIGNAL(keywordChanged(int)), this, SLOT(localKeywordChanged(int)));
         keywordWidgets_.push_back(w);
         ui_.ModuleControlStack->addWidget(w);
 
         // Map our button to the new stack page
         controlStackMap_[b] = ui_.ModuleControlStack->count() - 1;
 
-        currentGroupName = groupName;
         if (!firstButton)
             firstButton = b;
     }
```

Comment:
```suggestion
        if (!firstButton)
        {
            firstButton = b;
            firstButton->setChecked(true);
        }
    }
```

Better to put these together (remove the if statement after this)

Reply:
:+1: